src/test/scala/org/broadinstitute/dsde/workbench/leonardo/db/ClusterComponentSpec.scala

```diff
@@ -82,13 +82,10 @@ class ClusterComponentSpec extends TestComponent with FlatSpecLike {
       labels = Map.empty)
     dbFailure { _.clusterQuery.save(c4) } shouldBe a[SQLException]
 
-    dbFutureValue { _.clusterQuery.deleteByGoogleId(c1.googleId) } shouldEqual 1
-    dbFutureValue { _.clusterQuery.list() } shouldEqual Seq(c2)
```

Comment:
Restore these `list()` tests.  To make these pass, update `list()` to only return active clusters.

Reply:
I'm doing that as part of https://github.com/broadinstitute/leonardo/pull/21! Actually was thinking of making a new method `listActive()` which returns only active clusters.

Reply:
ok!